infra.bs

```diff
@@ -240,6 +240,40 @@ as well as the single algorithm would. Therefore performance is best left as a f
 over.
 
 
+<h3 id=algorithm-limits>Avoid limits on algorithm inputs</h3>
+
+<p>A document using the Infra Standard generally should not enforce specific limits on algorithm
+inputs with regards to their size, resource usage, or equivalent. This allows for competition among
+user agents and avoids constraining the potential computing needs of the future.
+
+<p tracking-vector>Nevertheless, user agents may impose <a>implementation-defined</a> limits on
+otherwise unconstrained inputs. E.g., to prevent denial of service attacks, to guard against running
+out of memory, or to work around platform-specific limitations.
+
+<p class=note>Global resource limits can used as side channels through a variant on a resource
```

Comment:
```suggestion
<p class=note>Global resource limits can be used as side channels through a variant on a resource
```
Nit: It looks like there is a missing 'be'; and perhaps also a missing comma before 'whereby' below?